content/applications/finance/accounting/fiscal_localizations/localizations/germany.rst

```diff
@@ -31,7 +31,134 @@ Export from Odoo to Datev
+   :alt: Upgrading Odoo Point of Sale from the Apps dashboard
+
+Register your company at the financial authority
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To register your company, go to :menuselection:`Settings --> General Settings --> Companies -->
+Update Info` and fill out the following fields:
+
+- **Company name**
+- Valid **address**
+- **VAT** number
+- **St.-Nr** (Steuernummer): this number is assigned by the tax office to every taxable natural or
+  legal person. (e.g., `2893081508152`)
+- **W-IdNr** (Wirtschafts-Identifikationsnummer): this number is used as a permanent
+  identification number for economically active persons.
+
+You can then **register your company through fiskaly** by opening the *fiskaly* tab and clicking on
+the *fiskaly Registration* button.
+
+.. image:: germany/fiskaly-registration.png
+   :align: center
+   :alt: Button to register a company through fiskaly in Odoo
+
+Once the registration has been finalized, new fields appear:
+
+- **fiskaly organization ID** refers to the ID of your company at the fiskaly side.
+- **fiskaly API key** and **secret** are the credentials the system uses to access the services
+  offered by fiskaly with your company.
+
+.. image:: germany/fiskaly-keys.png
+   :align: center
+   :alt: fiskaly keys as displayed on Odoo
+
+.. note::
+   It is possible to request new credentials if there is any issue with the current ones.
+
+Create and link a Technical Security System to your PoS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: germany/create-tss.png
+   :align: right
+   :alt: "Create TSS" option from a point of sale
+
+To use your point of sale in Germany, you first have to create a :abbr:`TSS (Technical Security
+System)` for it.
+
+To do so, go to :menuselection:`Point of Sale --> Configuration --> Point of Sale`, open the point
+of sale you want to edit, then check the box next to **Create TSS** and *Save*.
+
+.. image:: germany/tss-ids.png
+   :align: right
+   :alt: Example of TSS ID and Client ID from fiskaly on Odoo Point of Sale
+
+Once the creation of the TSS is successful, you can find your **TSS ID** and **Client ID** under the
+*fiskaly API* section.
+
+- **TSS ID** refers to the ID of your TSS at fiskaly's side
+- **Client ID** refers to your PoS but at fiskaly's side.
+
+DSFinV-K
+--------
+
+.. image:: germany/dsfinv-k-export.png
+   :align: right
+   :alt: Menu to export DSFinV-K
+
+Whenever you close a PoS session, the orders' details are sent to the :abbr:`DSFinV-K (Digitale
+Schnittstelle der Finanzverwaltung für Kassensysteme)` service of fiskaly.
+
+In case of an audit, you can export the data sent to DSFinV-K by going to :menuselection:`Point of
+Sale --> Orders --> DSFinV-k exports`.
+
+These fields are mandatory:
+
+- **Name**
+- **Start Datetime** (only export data with dates larger than or equal to the given start date)
+- **End Datetime** (export data with dates smaller than or equal to the given start date)
```

Comment:
```suggestion
- **Start Datetime** (only export data with dates larger than or equal to the given start date)
- **End Datetime** (only export data with dates smaller than or equal to the given end date)
```
Added _only_ for consistency, renamed _start_ date to _end_ date for End Datetime